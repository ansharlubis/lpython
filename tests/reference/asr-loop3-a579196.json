{
    "basename": "asr-loop3-a579196",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/loop3.py",
    "infile_hash": "498c7c40ca352ed21dde33c128aff4d59d3c329e770e3a2ff3eb3fad",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-loop3-a579196.stdout",
    "stdout_hash": "010ffe61c558debdf3193e2b2f4ab9d9070fe859c6849f40b4efab6c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}