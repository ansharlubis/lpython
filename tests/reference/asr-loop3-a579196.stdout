([1m[35mTranslationUnit[39m[0m 
   ([33mSymbolTable[39m 
      1 
      {
         test_pass: 
            ([1m[35mSubroutine[39m[0m 
               ([33mSymbolTable[39m 
                  2 
                  {
                     a: 
                        ([1m[35mVariable[39m[0m 
                           2 
                           a [1m[32m
                           Local[39m[0m () () [1m[32m
                           Default[39m[0m 
                           ([1m[35mInteger[39m[0m 4 []) [1m[32m
                           Source[39m[0m [1m[32m
                           Public[39m[0m [1m[32m
                           Required[39m[0m .false.)
                  }) 
               test_pass [] [
               ([1m[35m=[39m[0m 
                  ([1m[35mVar[39m[0m 2 [33ma[39m) ([1m[35mIntegerConstant[39m[0m [36m1[39m 
                  ([1m[35mInteger[39m[0m 4 [])) ()) 
               ([1m[35mWhileLoop[39m[0m 
                  ([1m[35mIntegerCompare[39m[0m 
                     ([1m[35mVar[39m[0m 2 [33ma[39m) [1m[32m
                     Gt[39m[0m ([1m[35mIntegerConstant[39m[0m [36m0[39m 
                     ([1m[35mInteger[39m[0m 4 [])) 
                     ([1m[35mLogical[39m[0m 4 []) ()) [])] [1m[32m
               Source[39m[0m [1m[32m
               Public[39m[0m [1m[32m
               Implementation[39m[0m () .false. .false.)
      }) 
   [])
(TranslationUnit (SymbolTable 1 {main_program: (Program (SymbolTable 3 {}) main_program [] []), test_pass: (Subroutine (SymbolTable 2 {a: (Variable 2 a Local () () Default (Integer 4 []) Source Public Required .false.)}) test_pass [] [(= (Var 2 a) (IntegerConstant 1 (Integer 4 [])) ()) (WhileLoop (IntegerCompare (Var 2 a) Gt (IntegerConstant 0 (Integer 4 [])) (Logical 4 []) ()) [])] Source Public Implementation () .false. .false.)}) [])
